packages/nc-gui/components/smartsheet/header/Cell.vue

```diff
@@ -1,6 +1,6 @@
 <script setup lang="ts">
 import type { ColumnReqType, ColumnType } from 'nocodb-sdk'
-import { ActiveViewInj, ColumnInj, IsFormInj, IsKanbanInj, inject, provide, ref, toRef, useUIPermission } from '#imports'
+import { IsFormInj, IsKanbanInj, IsLockedInj, inject, provide, ref, toRef, useUIPermission } from '#imports'
```

Comment:
seems you removed `ColumnInj` by mistake.

Reply:
fixed it , but removing that import didn't cause any error any idea why ?